site/en/blog/new-in-chrome-90/index.md

```diff
@@ -0,0 +1,277 @@
+visible.
+
+{% Columns %}
+{% Column %}
+
+```css
+.overflow-clip {
+  overflow: clip;
+  overflow-clip-margin: 25px;
+}
+```
+
+{% endColumn %}
+{% Column %}
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/GSyCxP6F48a9tJ0U5HAh.png", alt="Square box with text CSS is awesome, where awesome overflows out of the box", width="506", height="488" %}
+
+{% endColumn %}
+{% endColumns %}
+
+See `overflow: clip` in action at <https://petele-css-is-awesome.glitch.me/>
+
+## Feature Policy is now Permissions Policy {: #permission-policy}
+
+Back in Chrome 74, we introduced the [Feature Policy API][feature-policy-intro],
+which allows you to selectively enable, disable, and modify the behavior of
+certain APIs and web features in the browser. These policies are a contract
+between you and the browser. They inform the browser about what your intent is.
+
+If your code, or any of the 3rd party libraries you use violate your
+preselected rules, the browser overrides the behavior with better UX or
+just says - talk to the hand - blocking the API altogether.
+
+Starting in Chrome 90, the Feature Policy API will be renamed to the
+Permissions Policy, and the HTTP header has been renamed along with it. At
+the same time, the community has settled on a new syntax, based on Structured
+Field Values for HTTP.
+
+{% Columns %}
+{% Column %}
+
+### Chrome 90 and later
+
+```http
+Permissions-Policy: geolocation=()
+```
+
+{% endColumn %}
+{% Column %}
+
+### Chrome 89 and earlier
+
+```http
+Feature-Policy: geolocation 'none'
+```
+
+{% endColumn %}
+{% endColumns %}
+
+If you're interested in how to use this on your site, check out
+[Introduction to Feature Policy][feature-policy-intro].
+
+## Declarative Shadow DOM {: #declarative }
+
+[Shadow DOM][shadow-dom], part of the Web Components standard, provides a way
+to scope CSS styles to a specific DOM subtree and isolate that subtree from
+the rest of the document. Until now, the only way to use Shadow DOM was to
+construct a shadow root using JavaScript.
+
+```js
+const host = document.getElementById('host');
+const opts = {mode: 'open'};
+const shadowRoot = host.attachShadow(opts);
+const html = '<h1>Hello Shadow DOM</h1>';
+shadowRoot.innerHTML = html;
+```
+
+This works fine for client-side rendering, but not so well in server side
+rendering where there is no built in way to express Shadow Roots in the server
+generated HTML. But, starting in Chrome 90, with the Declarative Shadow DOM,
+you're good to go. You can create shadow roots using only HTML.
+
+A Declarative Shadow Root is a `<template>` element with a `shadowroot`
+attribute. It's detected by the HTML parser and immediately applied as the
+shadow root of its parent element.
+
+```html
+<host-element>
+  <template shadowroot="open">
+    <slot></slot>
+  </template>
+  <h2>Light content</h2>
+</host-element>
+```
+
+Loading the pure HTML markup results in this DOM tree.
+
+```html
+<host-element>
+  #shadow-root (open)
+  <slot>
+    ↳
```

Comment:
Should this be here?

Reply:
I think so, I copied it directly from Jason's article.